src/wallet/rpcdump.cpp

```diff
@@ -1446,21 +1450,27 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
         LOCK(pwallet->cs_wallet);
         EnsureWalletIsUnlocked(pwallet);
 
-        // Verify all timestamps are present before importing any keys.
+        const int64_t minimumTimestamp = 1;
```

Comment:
style-nit: Why is the comment and constant moved?

Reply:
Reverted.